src/sage/rings/polynomial/laurent_polynomial_ideal.py

```diff
@@ -196,9 +198,27 @@ def __contains__(self, f):
         if not f or f in self.gens():
             return True
         f = self.ring()(f)
-        g = f.__reduce__()[1][0]
+        if self.ring().ngens() > 1:
+            g = f.__reduce__()[1][0]
+        else:
+            g = f.__reduce__()[1][1]
```

Comment:
You made this change for a reason. Please add a doctest showing what was not working beforehand.

Reply:
I added a test for the two __reduce__ methods.

Reply:
Those doctests don't really test why you made this change. Something was broken: the containment check for univariate Laurent polynomials. As such, you need to add that test in showing the contains now works properly in that case.